western_hlad/description.md

```diff
@@ -1,18 +1,30 @@
-# Western (O.Hlad)
+# Modern (O.Hlad,1988)
```

Comment:
I think, when we say, the IAU-constellations are defined ~1930 which is almost a century now that there are many different styles of art since then. e.g. I did not really know "Hlad" before - in East Germany we often use the Czech variant from Prague (Rükl's lunar atlas!) but not this one. The name (Hlad) is apparently slavic and Czech was my first guess but I think it is an interesting contrast to Rey and S&T.

Reply:
At least in the context of the desktop edition, the year makes sense, as it's a "single, specific work" description.  I also knew Ceplecha-Rükl (even before S&T...), and Hlad was new to me until we received the contribution. Its author unfortunately did not write much, not even the complete first name.

Reply:
added the given names of the authors of this atlas 
and reworked the introductions